src/pages/contact.js

```diff
@@ -2,114 +2,138 @@ import React from "react"
+                method="post"
+                action="/success"
+                data-netlify="true"
+                data-netlify-honeypot="bot-field"
               >
-                City
-              </label>
-              <input
-                className="appearance-none block w-full bg-gray-200 text-gray-700 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
-                id="city"
-                type="text"
-                name="city"
-                placeholder="Manchester"
-              />
+                <input type="hidden" name="form-name" value="contact" />
+                <input type="hidden" name="bot-field" />
+                <div className="flex flex-wrap -mx-3 mb-6">
+                  <div className="w-full md:w-1/2 px-3 mb-6 md:mb-0 lg:w-full">
+                    <label
+                      className="block uppercase tracking-wide text-gray-700 text-xs font-bold mb-2"
+                      htmlFor="first-name"
+                    >
+                      Full Name *
+                    </label>
+                    <input
+                      className="appearance-none block w-full bg-gray-200 text-gray-700 border border-gray-200  py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
+                      id="first-name"
+                      name="firstName"
+                      type="text"
+                      placeholder="Please enter your name"
+                    />
+                  </div>
+                  <div className="w-full md:w-1/2 px-3 lg:w-full">
+                    <label
+                      className="block uppercase tracking-wide text-gray-700 text-xs font-bold 1 mb-2"
+                      htmlFor="last-name"
+                    >
+                      Email Address *
+                    </label>
+                    <input
+                      className="appearance-none block w-full bg-gray-200 text-gray-700 border border-gray-200  py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-500"
+                      id="last-name"
+                      name="lastName"
+                      type="text"
+                      placeholder="Please enter your email address"
+                    />
+                  </div>
+                </div>
+                <div className="flex flex-wrap -mx-3 mb-6">
+                  <div className="w-full px-3">
+                    <label
+                      className="block uppercase tracking-wide text-gray-700 text-xs font-bold 1 mb-2"
+                      htmlFor="message"
+                    >
+                      Message *
+                    </label>
+                    <textarea
+                      className="appearance-none block w-full bg-gray-200 text-gray-700 border border-gray-200  py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-500 h-40"
```

Comment:
Do focus ones after default styles. Just move h-40 to before the focus stuff here